cli_client/python/timesketch_cli_client/commands/sketch.py

```diff
@@ -53,3 +55,43 @@ def create_sketch(ctx, name, description):
         description = name
     sketch = api_client.create_sketch(name=name, description=description)
     click.echo(f"Sketch created: {sketch.name}")
+
+
+@sketch_group.command("add_event")
```

Comment:
Another thought here is: Do we want to have this in the sketch command? I see why you are doing that but it is a bit unintuitive. When I want to add an event I operate on a timeline and not the sketch. We may even want to be able to add event to other timelines in the future (separate discussion).

For the case of "manually added events" to the special timeline I would still argue that we want to have this ability in the timeline command.

Consider refactor.

WDYT?

Reply:
I had the same thought, on the contrary, if that special timeline is not yet existing so in the timeline command group it would be the only one not tied to a specific timeline (at least for the beginning).

Looking at the API, the method is nested on the sketch level and not on timeline (not saying that this bounds us to stick to it).

Looking at the Web UI, the button to add a manual event is also basically nested the sketch explore view (but close to the timelines).

But in the end I am fine with both:

``` timesketch timeline add_event —message „foobar“ —timestamp 123456 —timestamp_desc „test event“ 19 ``` where 19 would be the timeline ID for the manual events (if it is not or not yet existing we could err our)

And moving forward (if we want) we could open that command to other timelines as well (longterm).

We could also make:
``` timesketch events add —message „foobar“ —timestamp 123456 —timestamp_desc „test event“ —timeline_id 19 ```

So Make it part of the events group.